### ISSUE TITLE
Update existing Measures

### ISSUE THREAD

**Issue by borismichel** [borismichel/qs-mir#3]
If an Object Id is already present (or the app that is exported to is equal to the one that the object was originally taken from) the user should be prompted to choose to either update or append. Use `ApplyPatches `with the respective property to update the existing object. This will also update all dependencies

**Comment by veglar** [comment]
Yes this was one of the first issues I faced when trying QS MIR. 
When Importing measures from an master application and then trying to push changes to a development copy or a new application all Exports generate new duplicate measures. 

_Maybe this could be solved if the link between QS MIR and Qlik Sense measure where done on the Qlik Sense Master object ID._
